engine/lib/admin.php

```diff
@@ -196,7 +196,11 @@ function _elgg_create_notice_of_pending_upgrade($event, $type, $object) {
  */
 function _elgg_admin_init() {
 
-	elgg_register_event_handler('pagesetup', 'system', '_elgg_admin_pagesetup', 1000);
+	$url = elgg_get_simplecache_url('admin.css');
+	elgg_register_css('elgg.admin', $url);
+		
+	elgg_register_plugin_hook_handler('register', 'menu:admin_header', '_elgg_admin_header_menu');
```

Comment:
True. A mention in the upgrade docs would be sufficient. E.g. "Many core menu items are now added to the menu later, in the `register` hook. If your plugin looks for/alters core menu items, you may need to use the `register` hook with a higher priority."